frontend/src/pages/Landing/RegionalSelect.js

```diff
@@ -0,0 +1,156 @@
+/* eslint-disable react/jsx-props-no-spreading */
+import React, { useState } from 'react';
+import PropTypes from 'prop-types';
+import Select, { components } from 'react-select';
+import { Button } from '@trussworks/react-uswds';
+
+import 'uswds/dist/css/uswds.css';
+import '@trussworks/react-uswds/lib/index.css';
+import './index.css';
+
+import triangleDown from '../../images/triange_down.png';
+import check from '../../images/check.svg';
+
+const DropdownIndicator = (props) => (
+  <components.DropdownIndicator {...props}>
+    <img alt="" style={{ width: '22px' }} src={triangleDown} />
+  </components.DropdownIndicator>
+);
+
+const Placeholder = (props) => <components.Placeholder {...props} />;
+
+export const getUserOptions = (regions) => regions.map((region) => ({ value: region, label: `Region ${region}` }));
+
+const styles = {
+  container: (provided, state) => {
+    // To match the focus indicator provided by uswds
+    const outline = state.isFocused ? '0.25rem solid #2491ff;' : '';
+    return {
+      ...provided,
+      outline,
+    };
+  },
+  input: () => ({ display: 'none' }),
+  control: (provided) => ({
+    ...provided,
+    borderColor: '#0166AB',
+    backgroundColor: '#0166AB',
+    borderRadius: '5px',
+    paddingLeft: '5px',
+    paddingTop: '4px',
+    paddingBottom: '4px',
+    whiteSpace: 'nowrap',
+    color: 'white',
+    width: '120px',
+  }),
+  indicatorSeparator: () => ({ display: 'none' }),
+  menu: (provided) => ({
+    ...provided,
+    width: '200px',
+  }),
+  // placeholder: () => ({
+  //   color: 'white', fontWeight: 600, fontSize: '17px', marginRight: '-5px',
+  // }),
+  option: (provided, state) => ({
+    ...provided,
+    color: state.isSelected ? '#0166AB' : 'black',
+    fontWeight: state.isSelected ? '700' : 'normal',
+    backgroundColor: state.isSelected ? '#F8F8F8' : '#FFFFFF',
+    padding: 11,
+  }),
+  singleValue: (provided) => {
+    const single = { color: '#FFFFFF', fontWeight: 600 };
+
+    return {
+      ...provided, ...single,
+    };
+  },
+  valueContainer: () => ({ padding: '10px 8px' }),
+};
+
+function RegionalSelect(props) {
+  const {
+    regions, onApply,
+  } = props;
+
+  const [selectedItem, setSelectedItem] = useState();
+  const [appliedItem, setAppliedItem] = useState();
+  const [menuIsOpen, setMenuIsOpen] = useState(false);
+
+  // const delayedCloseMenu = () => setTimeout(setMenuIsOpen(false), 1000);
+
+  const CustomOption = (props1) => {
```

Comment:
I'm a bit confused by "props1." Does the 1 signify something?

Reply:
1 was used to distinguish between props. Ideally, we would want to move CustomOption out to avoid this. That was my plan.

Reply:
I did run into problems though with the move. Instead, I renamed props1 to better describe and possibly refactor that code later if needed.